tests/pdctl/scheduler/scheduler_test.go

```diff
@@ -691,47 +690,3 @@ func mightExec(re *require.Assertions, cmd *cobra.Command, args []string, v inte
 	}
 	json.Unmarshal(output, v)
 }
-
-func TestForwardSchedulerRequest(t *testing.T) {
```

Comment:
Why remove it?

Reply:
it is duplicated with TestScheduler, when TestScheduler support to run in two modes.